tests/test_helpers/test_commands.py

```diff
@@ -312,6 +312,28 @@ def test_localcommands_run(
     assert patched_subprocess.Popen.mock_calls == expected_calls
 
 
+@patch('invenio_cli.helpers.commands.subprocess')
+@patch('invenio_cli.helpers.docker_helper.subprocess')
+def test_localcommands_stop(
+        patched_docker_helper, patched_subprocess, fake_cli_config):
+    commands = LocalCommands(fake_cli_config)
+
+    # Needed to fake call to docker-compose --version but not call
```

Comment:
Doubt: What does this comment applies to?

Reply:
This comment was made when developing the `test_localcommands_destroy` and I can't really remember the reason why was done.